README.md

```diff
@@ -25,10 +25,8 @@ Example using standard AWS SDK credentials and an assumed role:
 #!/bin/bash
 set -ex
 docker run --rm -it \
-    -e ACCESS_KEY_ID=${ACCESS_KEY_ID:?}
-    -e SECRET_ACCESS_KEY=${SECRET_ACCESS_KEY:?}
-    -e ROLE_ARN=${ROLE_ARN:?}
     -e CLUSTER=your-cluster-name \
     -e AUTOSCALING_GROUPS=${AUTOSCALING_GROUP:?} \
+    -v "$HOME/.aws:/.aws"
```

Comment:
Can you push this to ecr, and then readme main local setup ye wala as it is chod de, and for everyone else, add a step like `./rotate-nodegroups.sh role1 role2` and this script will fetch asgs via roles and rotate those